NEWS.md

```diff
@@ -1,5 +1,11 @@
-## 2019-03-21 v1.3.0-SNAPSHOT
+## 2019-05-02 v1.3.0-SNAPSHOT
```

Comment:
Do we need to change a date here ?

Reply:
it doesn't really matter, we will update the date anyway to reflect the release date